geometry_msgs/Point point
---
bool success
int32 index
string message
